### PR TITLE
Quorum - Besu Hybrid network tests

### PR DESCRIPTION
**Pull request by jbhurat**
This PR introduces support QBFT Consensus and Hybrid Networks. It does the following;

- Creates a `typical-hybrid` network and run tests against it
- `basic-istanbul` test is updated to make it more clear
- Swap `networks/template:istanbul-*` with `networks/template:qbft-*`

### CODE REVIEW

File: .github/workflows/run.yml [hunk 1]
```diff
@@ -97,16 +97,19 @@ jobs:
         tag:
           - 'basic || basic-raft || (advanced && raft) || networks/typical::raft'
           - 'basic || basic-istanbul || (advanced && istanbul) || networks/typical::istanbul'
+          - 'basic || basic-istanbul || (advanced && istanbul) || networks/typical::qbft'
           - 'gcmode && block-sync && networks/template::raft-3plus1'
           - 'gcmode && block-sync && networks/template::istanbul-3plus1'
+          - 'gcmode && block-sync && networks/template::qbft-3plus1'
           - 'learner-peer-management || raftdnsenable && networks/template::raft-3plus1'
-          - 'validator-management && networks/template::istanbul-3plus1'
```
Comment (baptiste-b-pegasys) [inline .github/workflows/run.yml:103]: why removing istanbul ATs ?

Comment (jbhurat) [reply .github/workflows/run.yml:103]: We are swapping istanbul ATs with QBFT ATs

**Review by nmvalera — APPROVED** [review]
LGTM

**Review by trung — APPROVED** [review]
LGTM

**Pull request merged**